dev/src/components/CourseCard.tsx

```diff
@@ -43,10 +42,10 @@ const CourseCard = ({ course, enrollment }: CardProps): JSX.Element => {
                     <h5 className="card-title">{course.getName()} - {course.getTag()}/{course.getYear()}</h5>
                     {hasNone(status) ?
                         <div className="btn btn-primary course-button" onClick={() => actions.enroll(course.getId())}>Enroll</div>
-                    : hasPending(status) ?
-                        <div className="btn btn-secondary course-button disabled">Pending</div>
-                    :
-                        <div className="btn btn-primary course-button" onClick={() => history.push("/course/" + enrollment.getCourseid())}>Go to Course</div>
+                        : hasPending(status) ?
```

Comment:
Maybe the `?` and `:` could be moved first on each line, including the one after `hasNone(status)` so that it (hopefully) becomes a little easier to see where the different parts of the if-else construction.